### ISSUE TITLE
MobileNets: Efficient Convolutional Neural Networks for Mobile Vision Applications

### ISSUE THREAD

**Issue by sile** [sile/readings#66]
URL
-----

https://arxiv.org/abs/1704.04861

Abstract
----------

> We present a class of efficient models called MobileNets for mobile and embedded vision applications. MobileNets are based on a streamlined architecture that uses depth-wise separable convolutions to build light weight deep neural networks. We introduce two simple global hyper-parameters that efficiently trade off between latency and accuracy. These hyper-parameters allow the model builder to choose the right sized model for their application based on the constraints of the problem. We present extensive experiments on resource and accuracy tradeoffs and show strong performance compared to other popular models on ImageNet classification. We then demonstrate the effectiveness of MobileNets across a wide range of applications and use cases including object detection, finegrain classification, face attributes and large scale geo-localization. 

Table of Contents
--------------------

- (1) Introduction
- (2) Prior Work
- (3) MobileNet Architecture
  - (3-1) Depthwise Separable Convolution
  - (3-2) Network Structure and Training
  - (3-3) Width Multiplier: Thinner Models
  - (3-4) Resolution Multiplier: Reduced Representation
- (4) Experiments
  - (4-1) Model Choices
  - (4-2) Model Shrinking Hyperparameters
  - (4-3) Fine Grained Recognition
  - (4-4) Large Scale Geolocalization
  - (4-5) Face Attributes
  - (4-6) Object Detection
  - (4-7) Face Embeddings
- (5) Conclusion

BibTeX
------

```bibtex
@article{howard2017mobilenets,
  title={Mobilenets: Efficient convolutional neural networks for mobile vision applications},
  author={Howard, Andrew G and Zhu, Menglong and Chen, Bo and Kalenichenko, Dmitry and Wang, Weijun and Weyand, Tobias and Andreetto, Marco and Adam, Hartwig},
  journal={arXiv preprint arXiv:1704.04861},
  year={2017}
}
```
